### ISSUE TITLE
"Gradle in Debian does not support Zinc compiler" error

### ISSUE THREAD

**Issue by DwayneSmurdon** [gradle/gradle#10025]
**NOTE: Resolved by upgrading to Gradle 5.5.1**

### Expected Behavior
Compilation of Play 2.6 project.

### Current Behavior
Compilation failure.
Caused by: org.gradle.api.GradleException: Gradle in Debian does not support Zinc compiler.

Partial Stacktrace:
```
        at org.gradle.execution.taskgraph.DefaultTaskGraphExecuter$EventFiringTaskWorker.execute(DefaultTaskGraphExecuter.java:230)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.processTask(DefaultTaskPlanExecutor.java:123)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.access$200(DefaultTaskPlanExecutor.java:79)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:104)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:98)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.execute(DefaultTaskExecutionPlan.java:626)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionPlan.executeWithTask(DefaultTaskExecutionPlan.java:581)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$TaskExecutorWorker.run(DefaultTaskPlanExecutor.java:98)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
Caused by: org.gradle.api.GradleException: Gradle in Debian does not support Zinc compiler.
        at org.gradle.language.scala.internal.toolchain.DefaultScalaToolProvider.newCompiler(DefaultScalaToolProvider.java:54)
        at org.gradle.language.scala.tasks.PlatformScalaCompile.getCompiler(PlatformScalaCompile.java:58)
        at org.gradle.language.scala.tasks.AbstractScalaCompile.compile(AbstractScalaCompile.java:87)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:46)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:780)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:747)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:121)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:336)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:328)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:199)
        at org.gradle.internal.progress.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:110)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:110)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:92)
        ... 29 more

```
### Context
I am running Ubuntu under Windows 10 (WSL).  

I have java 11 installed, but specify java "1.8" in the config.

I extracted: https://github.com/playframework/play-java-streaming-example

ran "gradle runPlayBinary"

It produces the stacktrace above.

I then tried to add:

    play 'org.scala-sbt:zinc_2.12:1.3.0-M8'

to the dependencies section, but that did not change the outcome.

Attempting to compile/run with sbt now.

This is my first Gradle project, so perhaps I'm making a rookie mistake. 

### Your Environment

------------------------------------------------------------
Gradle 4.4.1
------------------------------------------------------------

Build time:   2012-12-21 00:00:00 UTC
Revision:     none

Groovy:       2.4.16
Ant:          Apache Ant(TM) version 1.10.5 compiled on March 28 2019
JVM:          11.0.2 (Oracle Corporation 11.0.2+9-LTS)
OS:           Linux 4.4.0-17134-Microsoft amd64

**NOTE: Resolved by upgrading to Gradle 5.5.1**

**Comment by DwayneSmurdon** [comment]
Resolved by upgrading to Gradle 5.5.1

**Comment by big-guy** [comment]
Thanks @DwayneSmurdon 

The version of Gradle distributed through Debian's official repositories isn't something we support.  There are custom patches that remove functionality and break compatibility with the official version of Gradle (available from https://gradle.org/install/).
